docs/self-managed/operational-guides/update-guide/860-to-870.md

```diff
@@ -59,3 +59,84 @@ In that case, we append the following records:
 - `ASSIGN`
 - `ASSIGNING`
 - `ASSIGNED`
+
+## Prefix migration
+
+If a custom prefix has been defined for Operate and Tasklist (for example, using `camunda.operate.elasticsearch.indexPrefix`), then data must be migrated manually to the new indices.
+
+### Make the Elasticsearch instance reachable
+
+The migration relies on communicating directly with Elasticsearch. The Elasticsearch instance must be reachable on `http://localhost:9200`.
+
+- **Kubernetes**: Port-forward your Elasticsearch instance to ensure it is reachable. In the following example, `camunda-platform` can be replaced by the release name:
```

Comment:
```suggestion
- **Kubernetes**: Port-forward your Elasticsearch instance to ensure it is reachable. In the following example, `camunda-platform` can be replaced by the service name:
```

Reply:
I wouldn't reference release name here, as it might be that the customer/user has deployed ES separate